mathics/core/symbols.py

```diff
@@ -402,18 +401,6 @@ def __new__(cls, name: str, sympy_dummy=None):
             # For example, this can happen with String constants.
 
             self.hash = hash((cls, name))
-
```

Comment:
Dummy symbols are something related to sympy and not with the basic Mathics data structures. It is not needed anymore.